### ISSUE TITLE
Bugs in palindrome_products_test.py (Python 3)

### ISSUE THREAD

**Issue by marcuslxxii** [exercism/python#1136]
Following lines are incorrect:
```
self.assertIn(set(factors), [{1, 9}, {3, 3}])
self.assertEqual(set(factors), {91, 99})
self.assertEqual(set(factors), {11})
self.assertEqual(set(factors), {913, 993})
self.assertEqual(set(factors), {101, 101})
```
Must be replaced by the following lines:
```
self.assertCountEqual([set(x) for x in factors], [{1, 9}, {3}])
self.assertEqual([set(x) for x in factors], [{91, 99}])
self.assertEqual([set(x) for x in factors], [{11}])
self.assertEqual([set(x) for x in factors], [{913, 993}])
self.assertEqual([set(x) for x in factors], [{101}])
```

**Comment by cmccandless** [comment]
[Canonical-data](https://github.com/exercism/problem-specifications/blob/c5fb622157c01b182aadcb8adb1a6cf84148b530/exercises/palindrome-products/canonical-data.json)

**Comment by cmccandless** [comment]
Fixed in #1069
